cmd/cli/pkg/standalone/containers.go

```diff
@@ -230,6 +230,14 @@ func CreateControllerContainer(ctx context.Context, dockerClient *client.Client,
 	if doNotTrack {
 		env = append(env, "DO_NOT_TRACK=1")
 	}
+	
+	// Pass proxy environment variables to the container if they are set
+	proxyEnvVars := []string{"HTTP_PROXY", "HTTPS_PROXY", "NO_PROXY", "http_proxy", "https_proxy", "no_proxy"}
+	for _, proxyVar := range proxyEnvVars {
+		if value := os.Getenv(proxyVar); value != "" {
```

Comment:
nit: `LookupEnv` may be a bit more accurate. Though I assume none of those variables make sense with an empty value.

Reply:
The way we did it is the curl_getenv way.

We need this PR unblocked, lets change